### DISCUSSION TITLE
admin password not working

### DISCUSSION THREAD

**Discussion by pcboysir** [balzack/databag#52] [Q&A]
hi
i set the user to ADMIN and password to 123456789
but on web I get the message  
login error
Please confirm your username and password.
I run this commands
sqlite3 /var/lib/databag/databag.db
select str_value from configs where config_id='token';
and show me the password is 123456789
I delete the Containers and set it up again whit different value for user and password and still that error
please help me

**Comment by balzack** [comment]
There are two different login screens.
If you click on the 'cog' in the upper right, it will take you to the admin login screen: 
https://<DOMAIN</#/admin

The admin password should work there. The other login screen is for the accounts you create on your server.

Let me know if this is the issue, other people have had this confusion. This is something I should change to make it more intuitive.

**Reply by pcboysir** [reply]
hi
thank you for your help
if you add this part 
If you click on the 'cog' in the upper right, it will take you to the admin login screen:
https://<DOMAIN</#/admin
to the installation help on git
nobody confuse like me
 i have some question 
1- if any user forget his/her password how can reset it ?
2 - in send file section only for picture video and audio please tell how to send another types of files  
3- there is any possibility to sync user from active directory or not ?

**Comment by balzack** [comment]
hello,

Thanks for the suggestion, I just added it to the Portainer instructions.

1] if a user forgets their password, you can click on the 'lock' icon in the admin dashboard. This will create an access link for the account. Through that link a user can access their account and change the password.
2] I don't yet support other file types, but it is a high priority feature request. What file types would you need?
3] Active directory is not supported, people are starting to request that integration, so the feature is increasing in priority.

Thank you for using this project!

**Reply by pcboysir** [reply]
hi
thank you for quick response
most file type in my opinion is
office file such as xlx doc and ....
archive file such as rar zip and ...
in fact if possible to  send any type of file is verry better

**Comment by balzack** [comment]
Okay thanks, so if I add a feature to attach a generic file, that would meet your needs?

**Reply by pcboysir** [reply]
yes 
thank you
